x/gravity/keeper/batch.go

```diff
@@ -114,7 +114,9 @@ func (k Keeper) OutgoingTxBatchExecuted(ctx context.Context, tokenContract types
 
 	totalToBurn := math.NewInt(0)
 	for _, tx := range b.Transactions {
-		totalToBurn = totalToBurn.Add(tx.Erc20Token.Amount)
+		// We need to burn the total amount transferred, as well as the total
```

Comment:
This is how it was before, I just added the tax. Since [this](https://github.com/palomachain/paloma/pull/964) got merged we burn the amount transferred. Do we want to keep the behaviour before that PR that only burns if it's an external token?

We also always [mint](https://github.com/palomachain/paloma/blob/master/x/gravity/keeper/attestation_handler.go#L98) when transferring to Paloma, and that would have to be revised as well.

Reply:
Thanks all!